data/src/main/kotlin/com/alfresco/content/data/AppConfigUtils.kt

```diff
@@ -0,0 +1,121 @@
+package com.alfresco.content.data
+
+import android.content.Context
+import android.os.Environment
+import com.google.common.reflect.TypeToken
+import com.google.gson.Gson
+import java.io.BufferedReader
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileReader
+import java.io.FileWriter
+import java.io.IOException
+import java.lang.StringBuilder
+
+const val APP_CONFIG_JSON = "app.config.json"
+
+/**
+ * @property timeStamp
+ * This method check if the given timeStamp has passed the 24 hours or not
+ */
+fun isTimeToFetchConfig(timeStamp: Long): Boolean {
+    val hours24 = (1 * 24 * 60 * 60 * 1000)
```

Comment:
hours24 > hoursInMiliseconds
24 declare as constant.

Reply:
Updated it.